### PR TITLE
Fixed the READMEs, did some reorganizing

### PR DESCRIPTION
**Pull request by stopyoukid**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,8 @@
+#Error Logs
+npm-debug.log
```
Comment (darthtrevino) [inline .gitignore]:
```
*-error.log*
*-debug.log*
```

File: README.md [hunk 2]
```diff
@@ -3,16 +3,15 @@
...
 
-## Building
-* Running `npm test` will do the following:
-  * Compiles the `src` directory.
-  * Creates a `.pbiviz` file in the `dist\powerbi` directory.
-    * Go to [Power BI](https://app.powerbi.com/), and to import your new visual.
+## Usage
+* Install [node.js 6+](https://nodejs.org)
+* Install [yarn](https://yarnpkg.com/lang/en/docs/install)
+* Run `yarn` on the project directory, which will install all the dependencies
```
Comment (darthtrevino) [inline README.md]: **in** the project directory

File: packages/tablesorter/src/providers/JSONDataProvider.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ import * as d3 from "d3";
 
 const debug = require("debug"); // tslint:disable-line
 const log = debug("essex:widget:tablesorter:JSONDataProvider");
-import ldIsNaN = require("lodash/isNaN");
+const ldIsNaN = require("lodash/lang/isNaN"); //tslint:disable-line
```
Comment (darthtrevino) [inline packages/tablesorter/src/providers/JSONDataProvider.ts:37]: Can you reference lodash internal functions using import if you install `@types/lodash`?

**Review by darthtrevino — APPROVED** [review]
Minor comments

File: packages/tablesorter/src/providers/JSONDataProvider.ts (hunk 3 above)
Comment (stopyoukid) [reply packages/tablesorter/src/providers/JSONDataProvider.ts:37]: My plan was to keep it the same as the line I removed, `import ldIsNan = require..` which works great.  But, I downgraded to `3.6.0`, cause that is what the app has been tested against, and it seems that the `@types/lodash` either doesn't support lodash 3, or they weren't defining the individual functions as modules until lodash 4.  Either way, typescript wasn't picking up the typings using the `import` style.

**Comment by stopyoukid** [comment]
#31 #30

**Pull request merged**